core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt

```diff
@@ -279,7 +279,7 @@ private fun buildZones(builder: RawInfraBuilder) {
     val nodeHasZone = BooleanArray(builder.getTrackNodes().size.toInt()) { false }
 
     // TODO: change this algorithm to add all nodes to zones separately from iterating on track
-    // sections
+    //       sections
```

Comment:
Oversight?

Reply:
Idea likes it better to highlight TODO 🤷